algorithms/kernel/adaboost/adaboost_model_fpt.cpp

```diff
@@ -38,17 +38,14 @@ namespace interface2
  * \DAAL_DEPRECATED_USE{ Model::create }
  */
 template <typename modelFPType>
-DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy) :
-    _nFeatures(nFeatures),
-    _models(new data_management::DataCollection())
+DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy) : _nFeatures(nFeatures), _models(new data_management::DataCollection())
 {
     _alpha = data_management::NumericTablePtr(new data_management::HomogenNumericTable<modelFPType>(NULL, 1, 0));
 }
 
 template <typename modelFPType>
-DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy, services::Status &st) :
-    _nFeatures(nFeatures),
-    _models(new data_management::DataCollection())
+DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy, services::Status & st)
+    : _nFeatures(nFeatures), _models(new data_management::DataCollection())
```

Comment:
why here we have initialization in one row, but in algorithms/kernel/adaboost/adaboost_model.cpp on separate rows?

Reply:
Because here it fits in one row, while not fits for another file.